python/taichi/ui/utils.py

```diff
@@ -64,8 +65,75 @@ def vec_to_euler(v):
     return yaw, pitch
 
 
+def try_get_wheel_tag(module):
+    try:
+        import wheel.metadata
+        wheel_path = f'{module.__path__[0]}-{".".join(map(str, module.__version__))}.dist-info/WHEEL'
+        meta = wheel.metadata.read_pkg_info(wheel_path)
+        return meta.get('Tag')
+    except Exception:
+        pass
+
+
+def try_get_loaded_libc_version():
+    assert platform.system() == "Linux"
+    with open('/proc/self/maps') as f:
+        content = f.read()
+
+    try:
+        import re
+        libc_path = next(v for v in content.split() if 'libc-' in v)
+        ver = re.findall(r'\d+\.\d+', libc_path)
+        if not ver:
+            return None
+        return tuple([int(v) for v in ver[0].split('.')])
+    except StopIteration:
+        return None
+
+
+class GGUINotAvailableException(Exception):
+    pass
+
+
 def check_ggui_availability():
     """Checks if the `GGUI` environment is available.
     """
-    if not _ti_core.GGUI_AVAILABLE:
-        raise Exception("GGUI Not Available")
+    if _ti_core.GGUI_AVAILABLE:
+        return
+
+    try:
+        # Try identifing the reason
+        import taichi
```

Comment:
Btw this might not pass the pylint check: https://docs.taichi-lang.org/docs/contributor_guide#enforce-code-style